core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/DummyProperties.scala

```diff
@@ -0,0 +1,16 @@
+package edu.uci.ics.texera.workflow.common.operators
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import com.fasterxml.jackson.annotation.JsonProperty
+class DummyProperties {
+  @JsonProperty
+  @JsonSchemaTitle("Dummy Attribute")
+  //@JsonPropertyDescription("Dummy Attribute for incompatible attribute")
```

Comment:
Please do not leave lines that are commented out. Either keep this or remove this.